js/views/pageNavVw.js

```diff
@@ -591,14 +611,175 @@ module.exports = baseVw.extend({
       Backbone.history.navigate(route, {trigger: true});
     });
   },
+  
```

Comment:
the part of searching through save data is our custom logic that wouldn't live in the plugin. A plugin should give you events / hooks where you could impart your own custom logic. A big thing we're looking for the plugin to give us is the common UI stuff, like keyboard navigation, which we wouldn't want to have to repeat for each area we want autosuggest.